assignments/objects.js

```diff
@@ -49,16 +106,45 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  "name": "Susan",
+  "age": 70,
+  "speak": function(){
+    return "Hello, my name is " + this.name + "!";
+  },
+
+   child : {
+    "name": "George",
+    "age": 50,
+    "speak": function(){
+      return "Hello, my name is " + this.name + "!";
+    },
+
+    grandchild : {
+      "name": "Sam",
+      "age": 30,
+      "speak": function(){
+        return "Hello, my name is " + this.name + "!";
+      },
+    }
+  }
+}
 
 // Log the parent object's name
+console.log(parent.name);
 
 // Log the child's age
+console.log(parent.child.age);
 
 // Log the name and age of the grandchild
+console.log(parent.child.grandchild.name);
+console.log(parent.child.grandchild.age);
```

Comment:
This is fine, but instead of doing two console.logs, you can separate with a comma:

console.log(parent.child.grandchild.name, parent.child.grandchild.age)